packages/plexis/src/index.js

```diff
@@ -1,6 +1,7 @@
 export {default as toPred, default as toPredecessor} from '@plexis/to-pred';
 export {default as toSucc, default as toSuccessor} from '@plexis/to-succ';
 export {default as toTitle, default as titleize} from '@plexis/to-title';
+export {default as toCamel, default as toCamelCase} from '@plexis/to-camel-case';
```

Comment:
According to the issue you shall export 3 methods `toCamelCase, camelize, camelCase` #14